sds_data_manager/lambda_code/SDSCode/batch_starter.py

```diff
@@ -450,16 +471,24 @@ def lambda_handler(event: dict, context):
             return
 
         # Start Batch Job execution for each instrument
-        for instrument in downstream_instruments_to_process:
-            filename = instrument["filename"]
-
-            batch_client.submit_job(
-                jobName=filename,
+        for downstream_data in downstream_instruments_to_process:
+            filename = downstream_data["filename"]
+
+            command = downstream_data["prepared_data"]
+            logger.info(f"Submitting job with this command - {command}")
+            # NOTE: The batch job name should contain only
+            # alphanumeric characters and hyphens.
+            logger.info(f"Job name: {instrument}-{level}-job")
```

Comment:
Does the job name need to be unique with an extra hash or anything added?

You might want to bring that string out and store it for logging here and setting it later.
```python
job_name = f"{instrument}-{level}-job"
logger.info("Job name: %s", job_name)
```